src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Another thing to note is the standard for React components are to suffix your files with `.jsx` rather than `.js`